triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -408,7 +435,8 @@ class Runner(
       offset = Array(acsResponses: _*).lastOption
         .fold(LedgerOffset().withBoundary(LedgerOffset.LedgerBoundary.LEDGER_BEGIN))(resp =>
           LedgerOffset().withAbsolute(resp.offset))
-    } yield (acsResponses.flatMap(x => x.activeContracts), offset)
+      activeContracts = acsResponses.flatMap(x => x.activeContracts)
```

Comment:
Using `flatMap` inside the for comprehension makes it look like we might be using the effects of the Future when you are really using `flatMap` on lists. Keeping this in `yield` makes the separation much clearer.

Reply:
Ok